src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php

```diff
@@ -179,6 +181,8 @@ public function process(ContainerBuilder $container)
         }
 
         $container->getDefinition($this->resolverServiceId)
-            ->replaceArgument(0, ServiceLocatorTagPass::register($container, $controllers));
+            ->replaceArgument(0, $controllerLocatorRef = ServiceLocatorTagPass::register($container, $controllers));
+
+        $container->setAlias($this->controllerLocator, (string) $controllerLocatorRef)->setPrivate(true);
```

Comment:
Being private is already the default, isn't it?

Reply:
Oops, indeed.